packages/docs/pages/button.tsx

```diff
@@ -1,145 +1,221 @@
-                }
-
-                useEffect(() => {
-                  if (isLoading) {
-                    simulateNetworkRequest().then(() => {
-                      setLoading(false);
-                    });
-                  }
-                }, [isLoading]);
-
-                const handleClick = () => setLoading(true);
-
-                return (
-                  <Button variant="primary" isLoading={isLoading} onClick={!isLoading ? handleClick : () => null}>
-                    Click to load
-                  </Button>
-                );
-              }}
-              {/* jsx-to-string:end */}
-            </CodePreview>
-          </Panel>
-          <Panel header="Disabled">
-            <Text>
-              A disabled state is used to indicate no action can be done using the button by passing a{' '}
-              <Code primary>disabled</Code> prop.
-            </Text>
-
-            <CodePreview>
-              {/* jsx-to-string:start */}
-              <>
-                <Button disabled>Disabled</Button>
-                <Button disabled variant="secondary">
-                  Disabled
-                </Button>
-                <Button disabled variant="subtle">
-                  Disabled
-                </Button>
-              </>
-              {/* jsx-to-string:end */}
-            </CodePreview>
-          </Panel>
-          <Panel header="Icons">
-            <Text>
-              A button can also include icons on either side of the text (or both). When using{' '}
-              <Code primary>iconOnly</Code>, the <Code primary>iconLeft</Code> &amp; <Code primary>iconRight</Code>{' '}
-              components will be removed.
-            </Text>
-
-            <CodePreview>
-              {/* jsx-to-string:start */}
-              <>
-                <Button iconOnly={<AddIcon title="add" />} />
-                <Button iconLeft={<AddIcon />}>Label</Button>
-                <Button iconLeft={<AddIcon />} iconRight={<ArrowDropDownIcon />}>
-                  Label
-                </Button>
-                <Button iconRight={<ArrowDropDownIcon />}>Label</Button>
-              </>
-              {/* jsx-to-string:end */}
-            </CodePreview>
-          </Panel>
-        </>
-      ),
-    },
-    {
-      id: 'props',
-      title: 'Props',
-      render: () => <ButtonPropTable inheritedProps={<MarginPropTable collapsible />} />,
-    },
-  ];
-
   return (
     <>
       <H1>Button</H1>
 
-      <PageNavigation items={items} />
+      <Panel header="Overview" headerId="overview">
+        <Text>
+          <Code primary>Buttons</Code> trigger an immediate action, which can continue within the current page, a new
+          page or modal.
+        </Text>
+        <Text bold>When to use:</Text>
+        <List>
+          <List.Item>
+            Use <Code primary>Primary Buttons</Code> for the most important action on a page (e.g. creating an object,
```

Comment:
I like where your head is at here, but we probably don't want this all to be in a `<Code>` element. The general rule of thumb is:
- `<Code primary />` for elements/prop names.
- `<Code />` for prop values.
(we have been/are inconsistent)

I'd change this to the following since "buttons" not entirely referring to the component name. Same applies for the rest of the page.
```suggestion
            Use <Code>primary</Code> buttons for the most important action on a page (e.g. creating an object,
```